tools/get_mnist_webarchive.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "Downloading mnist datasets from web.archive.org"
```

Comment:
This is a reasonable this to make the default. Please create a separate PR which changes urlPrefix in tensorflow-mnist-input-data/Setup.hs to be web.archive.org. Then this file should be reverted.